.github/workflows/release.yml

```diff
@@ -34,6 +34,16 @@ jobs:
         - name: Install dependencies
           run: poetry install
 
+        - name: Check if release is from main
+          run: |
+            if [[ "${{ github.ref }}" == *"test"* ]]; then
+              echo "This release is a test release."        
+              exit 0
+            fi
+            
+            echo "Checking out matching trader versions. In case you want to skip this check add 'test' to the tag name. Ex. v0.1.0-rc50test"
```

Comment:
Is this tag naming approved by @truemiller ?

Reply:
i believe this tag will break the cli's release process.. 
i think it's more strict than on javascript side. Yuri had tested this last week, and it required a full bump..

however, we could adopt a more robust versioning approach here, and use `dev` tags for tests moving forward, and we're currently just spamming `rc` versions

see: https://peps.python.org/pep-0440/#public-version-identifiers